src/script.js

```diff
@@ -3,25 +3,46 @@ import googleAnalytics from './analytics'
 import syncLibrary from './sync-library'
 
 // Replace layerStyles and textLayerStyles in the document with selected theme library
-export default function() {
-  const libraries = sketch.getLibraries().filter(l => l.valid && l.enabled)
-  const [, selectionIndex, ok] = sketch.UI.getSelectionFromUser(
-    "Select a theme library to replace magically document's layer styles and symbols 🎉",
-    libraries.map(l => l.name)
-  )
-  googleAnalytics('Open Camilo', 'Alert', 'UI')
-
-  // Depending selected control, current document will sync with predefined brand
-  if (ok) {
-    const doc = sketch.getSelectedDocument()
-    const chosenLibrary = libraries[selectionIndex]
-
-    googleAnalytics('Camilo replacement with', chosenLibrary.name, 'Library')
-
-    syncLibrary(doc, chosenLibrary)
+export function getOptionSelected(libraries) {
+  
+  const options = [];
+  var optionSelected = NSComboBox.alloc().initWithFrame(NSMakeRect(0, 0, 200, 25));
```

Comment:
let's try to keep a consistent coding style: use `const` instead of `var`, don't put `;`, etc. Eslint and Prettier should help with that, try to install plugins for them in your code editor

Reply:
I really appreciate these comments, ok, thanks!